.github/workflows/create-image-without-release.yml

```diff
@@ -0,0 +1,172 @@
+# This workflow will release project with Gradle
+name: Create Image Only (No Release)
+
+on:
```

Comment:
Why on push?

Reply:
it was for testing the workflow. It is temp change and will change it to workflow_dispatch